src/test/java/uk/gov/register/functional/DataDownloadFunctionalTest.java

```diff
@@ -29,7 +29,7 @@
 public class DataDownloadFunctionalTest {
 
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
```

Comment:
:ok_man: so can we change `@Rule` to `@ClassRule` in: `DeleteRegisterDataFunctionalTest`?